lib/lbt/bundle/Builder.js

```diff
@@ -411,7 +411,9 @@ class BundleBuilder {
 				try {
 					fileContent = JSON.stringify( JSON.parse( fileContent) );
 				} catch (e) {
-					log.error(e);
+					log.verbose("Ignoring compression of JSON content. " +
```

Comment:
I would state this in the reverse order, first the reason, then the consequences:

 Failed to parse JSON .... Ignoring error, skipping compression.

Reply:
done with: a63ccd6